package.json

```diff
@@ -54,19 +56,19 @@
         "husky": "^1.0.1",
         "react-dev-utils": "^8.0.0",
         "storybook-addon-jsx": "^7.0.0",
+        "styled-jsx": "^3.2.1",
```

Comment:
Yarn doesn't install peer dependencies, so we need to state it as a dev dependency to get local development to work, both for Storybook but also for running the build process.